ros/src/tl_detector/light_classification/tl_classifier.py

```diff
@@ -2,10 +2,30 @@
 import cv2
 import numpy as np
 
+import tensorflow as tf
+from helpers import load_graph
+
+import os.path as path
+
+# Load frozen graph of trained model
+five_up = path.abspath(path.join(__file__, '../../../../..'))
+SSD_INCEPTION_SIM = path.join(five_up, 'models/ssd_inception_v2_alex_sim/frozen_inference_graph.pb')
+detection_graph = load_graph(SSD_INCEPTION_SIM)
+
+# Get tensors
+# Input placeholder for the image
+image_tensor = detection_graph.get_tensor_by_name('image_tensor:0')
+# Each box represents a part of the image where a particular object was detected
+detection_boxes = detection_graph.get_tensor_by_name('detection_boxes:0')
+# Each score represent the level of confidence for each of the objects
+detection_scores = detection_graph.get_tensor_by_name('detection_scores:0')
+# The classification of the object (integer id)
+detection_classes = detection_graph.get_tensor_by_name('detection_classes:0')
+
 class TLClassifier(object):
     def __init__(self):
-        #TODO load classifier
-        pass
+	tf.reset_default_graph()
```

Comment:
Suggestion:
Load the tf model within the init file

Reply:
yes

Reply:
Done.